harness/determined/common/experimental/determined.py

```diff
@@ -202,4 +202,5 @@ def get_model_labels(self) -> List[str]:
         """
         r = self._session.get("/api/v1/model/labels")
 
-        return r.json().get("labels")
+        labels = r.json().get("labels")
+        return [str(label) for label in labels]
```

Comment:
since you already expect it to be a list of strings, you probably could do `return cast(List[str], labels)` which should be enough to satisfy the type checker.

Reply:
👍